src/app/services/constraintService.spec.ts

```diff
@@ -77,23 +128,86 @@ function evaluateNodeConstraintWithTwoRemovalCriteria() {
 }
 
 function evaluateCriterias() {
-  it('should evaluate criterias when it is passed one criteria that is false', () => {
-    const criterias = [criteria1];
-    spyOn(dataService, 'isCompleted').and.returnValue(false);
-    expect(service.evaluateCriterias(criterias)).toEqual(false);
+  describe('evaluateCriterias()', () => {
+    it('should evaluate criterias when it is passed one criteria that is false', () => {
+      const criterias = [criteria1];
+      spyOn(dataService, 'isCompleted').and.returnValue(false);
+      expect(service.evaluateCriterias(criterias)).toEqual(false);
+    });
+    it('should evaluate criterias when it is passed one criteria that is true', () => {
+      const criterias = [criteria1];
+      spyOn(dataService, 'isCompleted').and.returnValue(true);
+      expect(service.evaluateCriterias(criterias)).toEqual(true);
+    });
+    it('should evaluate criterias when it is passed multiple criterias and one is false', () => {
```

Comment:
I think we should say what we expect the function to return to make the test clearer
```suggestion
    it('should return false when it is passed multiple criterias and one is false', () => {
```